src/heretic/p_spec.c

```diff
@@ -835,6 +835,8 @@ void P_ShootSpecialLine(mobj_t * thing, line_t * line)
     }
 }
 
+void P_SetCenterMessage(player_t * player, const char *message);
```

Comment:
This does not belong here, instead it belongs in some header so that both this use and the actual definition (apparently in `p_inter.c`) are checked for consistency by the compiler.

Reply:
That's what i was going to do but where? The problem was introduced in the recent PR https://github.com/fabiangreffrath/crispy-doom/pull/555 and has nothing to do with TDM-GCC but while i was at it...

Reply:
Done